src/lib/comms/sol-network-util.h

```diff
@@ -55,10 +55,10 @@ sol_bluetooth_is_addr_str(const char *str)
     const char *p = str;
 
     while (*str) {
-        if (!isxdigit(*str++))
+        if (!isxdigit((int)*str++))
             return false;
 
-        if (!isxdigit(*str++))
+        if (!isxdigit((int)*str++))
```

Comment:
I didn't explain myself well, I was referring to @cabelitos comment about merging with an earlier commit.